PcapDotNet/src/PcapDotNet.Packets/Dhcp/Options/DhcpBootfileNameOption.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PcapDotNet.Packets.Dhcp.Options
+{
+    /// <summary>
+    /// RFC 2132.
+    /// This option is used to identify a bootfile when the 'file' field in
+    /// the DHCP header has been used for DHCP options.
+    /// <pre>
+    ///  Code  Len    Bootfile name
+    /// +-----+-----+-----+-----+-----+---
+    /// | 67  |  n  |  c1 |  c2 |  c3 | ...
+    /// +-----+-----+-----+-----+-----+---
+    /// </pre>
+    /// </summary>
+    public class DhcpBootfileNameOption : DhcpStringOption
+    {
+        /// <summary>
+        /// create new DhcpBootfileNameOption
+        /// </summary>
+        /// <param name="bootfileName">Bootfilename</param>
+        public DhcpBootfileNameOption(string bootfileName) : base(bootfileName, DhcpOptionCode.BootfileName)
```

Comment:
Well, in general in Pcap.Net, it's attempted to have the names readable so they can be used without being familiar with the RFC.
In this case, if "bootfile" is considered a word, this makes sense. Otherwise it makes more sense to have bootFilename.